gruntfile.js

```diff
@@ -10,7 +10,14 @@ module.exports = function(grunt)
                     basePath: 'themes',
                     declaration: false
                 },
-                src: ['src/default/assets/js/src/**/*.ts'],
+                src: [
+                    'src/default/assets/js/src/lib/**/*.ts',
+                    'src/default/assets/js/src/typedoc/Application.ts',
+                    'src/default/assets/js/src/typedoc/components/**/*.ts',
+                    'src/default/assets/js/src/typedoc/services/**/*.ts',
+                    'src/default/assets/js/src/typedoc/utils/**/*.ts',
```

Comment:
it was a single glob before. that is why it was not working. when using globbing the concatenation order can not be specified. but the code relies on some order.

one way to specify the order might be to use `/// reference` comments in the `.ts` files. i don't know the exact order the files might need to be loaded. from debugging i just saw that components and services need to be executed before the application is initialized. so an alternative globbing might be `src/default/assets/js/src/lib/**/*.ts`, `src/default/assets/js/src/typedoc/**/*.ts`, `src/default/assets/js/src/~bootstrap.ts`.

i was also wondering what the following option does.

    basePath: 'themes',

i could not find any documentation about it.

all client js is currently not working without changing concat order on my system. does it work on yours?